tfhe/benches/integer/bench.rs

```diff
@@ -849,52 +863,27 @@ criterion_group!(
 
 criterion_group!(misc, full_propagate, full_propagate_parallelized);
 
-// User-oriented benchmark group.
-// This gather all the operations that a high-level user could use.
-criterion_group!(
-    fast_integer_benchmarks,
-    bitand_parallelized,
-    bitnot_parallelized,
-    bitor_parallelized,
-    bitxor_parallelized,
-    add_parallelized,
-    sub_parallelized,
-    mul_parallelized,
-    neg_parallelized,
-    min_parallelized,
-    max_parallelized,
-    eq_parallelized,
-    ne_parallelized,
-    lt_parallelized,
-    le_parallelized,
-    gt_parallelized,
-    ge_parallelized,
-    left_shift_parallelized,
-    right_shift_parallelized,
-    rotate_left_parallelized,
-    rotate_right_parallelized,
-    scalar_add_parallelized,
-    scalar_sub_parallelized,
-    scalar_mul_parallelized,
-    scalar_left_shift_parallelized,
-    scalar_right_shift_parallelized,
-    scalar_eq_parallelized,
-    scalar_ne_parallelized,
-    scalar_lt_parallelized,
-    scalar_le_parallelized,
-    scalar_gt_parallelized,
-    scalar_ge_parallelized,
-    scalar_min_parallelized,
-    scalar_max_parallelized,
```

Comment:
did you check we cover the same functions as before ?

Reply:
Yes, I've just rearranged the current coverage into separate groups.

Reply:
ok thanks !